src/test/kotlin/no/nav/helse/OmsorgspengesoknadProsesseringWithMocks.kt

```diff
@@ -25,19 +25,19 @@ class OmsorgspengesoknadProsesseringWithMocks {
                 .stubLagreDokument()
                 .stubSlettDokument()
 
-            val kafkaEnvironment = KafkaWrapper.bootstrap()
+            val kafkaContainer = KafkaWrapper.bootstrap()
 
             val testArgs = TestConfiguration.asMap(
```

Comment:
Denne mock-klassen feiler nå med 
`Exception in thread "main" java.lang.IllegalStateException: Client[azure-v2] må være satt opp.`

Tipper det er p.g.a. problemet med HOCON cli parameter som vi snacket om tidigare.
Får se på det separat eller vente på att ktor kommer med fiks.